actionpack/lib/action_dispatch/middleware/templates/rescues/actionable_error.html.erb

```diff
@@ -0,0 +1,65 @@
+<header>
+  <h1>
+    <%= @exception.class.to_s %>
+    <% if @request.parameters['controller'] %>
+      in <%= @request.parameters['controller'].camelize %>Controller<% if @request.parameters['action'] %>#<%= @request.parameters['action'] %><% end %>
```

Comment:
Can this be simplified to? `[ "#{@request.parameters['controller'].camelize}Controller", @request.parameters['action'] ].compact.join("#") %>`